_sql-and-ppl/ppl/commands/stats.md

```diff
@@ -0,0 +1,477 @@
+* `PERC<percent>`/`P<percent>` -- Percentile shortcut functions
+* `MEDIAN` -- 50th percentile
+* `EARLIEST` -- Earliest value by timestamp
+* `LATEST` -- Latest value by timestamp
+* `FIRST` -- First non-null value
+* `LAST` -- Last non-null value
+* `LIST` -- Collect all values into array
+* `VALUES` -- Collect unique values into sorted array  
+  
+<!-- temporarily commented out because the functions section is not ported
+
+For detailed documentation of each function, see [Functions]({{site.url}}{{site.baseurl}}/sql-and-ppl/functions/#aggregate).
+
+-->
+
+## Example 1: Calculate the count of events  
+
+The following query calculates the count of events in the `accounts` index:
+  
+```sql
+source=accounts
+| stats count()
+```
+{% include copy.html %}
+  
+The query returns the following results:
+  
+| count() |
+| --- |
+| 4 |
+  
+
+## Example 2: Calculate the average of a field  
+
+The following query calculates the average age for all accounts:
+  
+```sql
+source=accounts
+| stats avg(age)
+```
+{% include copy.html %}
+  
+The query returns the following results:
+  
+| avg(age) |
+| --- |
+| 32.25 |
+  
+
+## Example 3: Calculate the average of a field by group  
+
+The following query calculates the average age for all accounts, grouped by gender:
+  
+```sql
+source=accounts
+| stats avg(age) by gender
+```
+{% include copy.html %}
+  
+The query returns the following results:
+  
+| avg(age) | gender |
+| --- | --- |
+| 28.0 | F |
+| 33.666666666666664 | M |
+  
+
+## Example 4: Calculate the average, sum, and count of a field by group  
+
+The following query calculates the average age, sum of ages, and count of events for all accounts, grouped by gender:
+  
+```sql
+source=accounts
+| stats avg(age), sum(age), count() by gender
+```
+{% include copy.html %}
+  
+The query returns the following results:
+  
+| avg(age) | sum(age) | count() | gender |
+| --- | --- | --- | --- |
+| 28.0 | 28 | 1 | F |
+| 33.666666666666664 | 101 | 3 | M |
+  
+
+## Example 5: Calculate the maximum of a field  
+
+The following query calculates the maximum age for all accounts:
+  
+```sql
+source=accounts
+| stats max(age)
+```
+{% include copy.html %}
+  
+The query returns the following results:
+  
+| max(age) |
+| --- |
+| 36 |
+  
+
+## Example 6: Calculate the maximum and minimum of a field by group  
+
+The following query calculates the maximum and minimum ages for all accounts, grouped by gender:
+  
+```sql
+source=accounts
+| stats max(age), min(age) by gender
+```
+{% include copy.html %}
+  
+The query returns the following results:
+  
+| max(age) | min(age) | gender |
+| --- | --- | --- |
+| 28 | 28 | F |
+| 36 | 32 | M |
+  
+
+## Example 7: Calculate the distinct count of a field  
+
+To retrieve the count of distinct values of a field, you can use `DISTINCT_COUNT` (or `DC`) function instead of `COUNT`. The following query calculates both the count and the distinct count of the `gender` field for all accounts:
```

Comment:
```suggestion
To retrieve the count of distinct values of a field, you can use the `DISTINCT_COUNT` (or `DC`) function instead of `COUNT`. The following query calculates both the count and the distinct count of the `gender` field for all accounts:
```